helm/stacks-blockchain/values.yaml

```diff
@@ -0,0 +1,360 @@
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  
+  ## Set to true for verbose logging. Set to false for normal verbosity
+  ##
+  debug: false
+
+  ## Set to true to enable json logging. Set to false for normal logging
+  ##
+  jsonLogging: false
+
+  ## Overriding these is typically not necessary, unless you wish to run the node in a different way
+  ##
+  command:
+    - /bin/stacks-node
+  args:
+    - start
+    - --config
+    - /src/stacks-node/config.toml
+
+  ## Extra environment variables to add to the node's container
+  ##
+  extraEnv: []
+    # - name: FOO
+    #   value: "bar"
+
+  ## Specify resource limits and requests for the node's container
+  ##
+  resources: {}
+    # limits:
+    #   cpu: 500m
+    #   memory: 512Mi
+    # requests:
+    #   cpu: 250m
+    #   memory: 128Mi
+
+  ## Rollback limit
+  ##
+  revisionHistoryLimit: 10
+
+  # The update strategy to apply
+  ##
+  updateStrategy: {}
+  #  rollingUpdate:
+  #    maxUnavailable: 1
+  #  type: RollingUpdate
+
+  # minReadySeconds to avoid killing pods before it's ready
+  ##
+  minReadySeconds: 0
+
+  ## Node labels for node pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: []
+  #  - key: "key"
+  #    operator: "Equal|Exists"
+  #    value: "value"
+  #    effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+  ## Affinity and anti-affinity
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+  # # An example of preferred pod anti-affinity, weight is in the range 1-100
+  # podAntiAffinity:
+  #   preferredDuringSchedulingIgnoredDuringExecution:
+  #   - weight: 100
+  #     podAffinityTerm:
+  #       labelSelector:
+  #         matchExpressions:
+  #         - key: app.kubernetes.io/name
+  #           operator: In
+  #           values:
+  #           - stacks-blockchain
+  #       topologyKey: kubernetes.io/hostname
+
+  # # An example of required pod anti-affinity
+  # podAntiAffinity:
+  #   requiredDuringSchedulingIgnoredDuringExecution:
+  #   - labelSelector:
+  #       matchExpressions:
+  #       - key: app.kubernetes.io/name
+  #         operator: In
+  #         values:
+  #         - stacks-blockchain
+  #     topologyKey: "kubernetes.io/hostname"
+
+  ## Total time it takes for the Container to stop normally
+  ##
+  terminationGracePeriodSeconds: 60
+  
+  ## Additional volumes for the node
+  ##
+  volumes: []
+  # - name: stacks-node-chain
+  #   persistentVolumeClaim:
+  #     claimName: stacks-node-chain
+
+  ## Additional volumeMounts for the node
+  ##
+  volumeMounts: []
+  # - name: stacks-node-chain
+  #   mountPath: /root/stacks-node
+
+  ## Additional containers to run alongside the node. Useful if adding a sidecar
```

Comment:
I think here we shouldn't use the `sidecar` term, since it's named "follower" in the docs. 
To us, we understand a sidecar to mean a follower, but sidecar is also a term used in k8s so it could cause some confusion

Reply:
Because this is a public chart, the term here is meant in the form of the latter, where it's simply a container running on the side of the main container, meant to help in some way. It's not intended to be thought of as the internal PBC sense of the word (which we're moving away from using anyways). I figured it would be appropriate here since this section is for extra containers to run on the side.